docs/IE_PLUGIN_DG/CompiledModel.md

```diff
@@ -0,0 +1,88 @@
+
+@snippet src/compiled_model.cpp compiled_model:ctor
+
+The implementation `compile_model()` is fully device-specific.
+
+### compile_model()
+
+The function accepts a const shared pointer to `ov::Model` object and applies OpenVINO passes using `transform_model()` function, which defines plugin-specific conversion pipeline. To support low precision inference, the pipeline can include Low Precision Transformations. These transformations are usually hardware specific. You can find how to use and configure Low Precisions Transformations in [Low Precision Transformations](@ref openvino_docs_OV_UG_lpt) guide.
+
+@snippet src/compiled_model.cpp compiled_model:compile_model
+
+> **NOTE**: After all these steps, the backend specific graph is ready to create inference requests and perform inference.
+
+### export_model()
+
+The implementation of the method should write all data to the `model_stream`, which is required to import a backend specific graph later in the `Plugin::import_model` method:
+
+@snippet src/compiled_model.cpp compiled_model:export_model
+
+### create_sync_infer_request()
+
+The method creates an synchronous inference request and returns it.
+
+@snippet src/compiled_model.cpp compiled_model:create_sync_infer_request
+
+While the public OpenVINO API has a single interface for inference request, which can be executed in synchronous and asynchronous modes, a plugin library implementation has two separate classes:
+
+- [Synchronous inference request](@ref openvino_docs_ie_plugin_dg_infer_request), which defines pipeline stages and runs them synchronously in the `infer` method.
+- [Asynchronous inference request](@ref openvino_docs_ie_plugin_dg_async_infer_request), which is a wrapper for a synchronous inference request and can run a pipeline asynchronously. Depending on a device pipeline structure, it can has one or several stages:
+   - For single-stage pipelines, there is no need to define this method and create a class derived from ov::IAsyncInferRequest. For single stage pipelines, a default implementation of this method creates ov::IAsyncInferRequest wrapping a synchronous inference request and runs it asynchronously in the `m_request_executor` executor.
+   - For pipelines with multiple stages, such as performing some preprocessing on host, uploading input data to a device, running inference on a device, or downloading and postprocessing output data, schedule stages on several task executors to achieve better device use and performance. You can do it by creating a sufficient number of inference requests running in parallel. In this case, device stages of different inference requests are overlapped with preprocessing and postprocessing stage giving better performance.
+   > **IMPORTANT**: It is up to you to decide how many task executors you need to optimally execute a device pipeline.
+
+### create_infer_request()
```

Comment:
it's not rendered well 
![Uploading Screenshot 2023-03-17 at 12.02.12 AM.png…]()